site/pom.xml

```diff
@@ -12,6 +12,10 @@
 		<version>0.8.0-SNAPSHOT</version>
 	</parent>
 
+	<properties>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
```

Comment:
I'll change the job name if you merge the commit. PLEASE? /me has been waiting for days now. :(